flake8_jungle/rules/jg06_function_too_long.py

```diff
@@ -27,6 +27,10 @@ def run(self, node: FunctionType) -> Iterable[Issue]:
             return issues
         length = node.end_lineno - node.lineno
 
+        docstring: str = ast.get_docstring(node, clean=False)
```

Comment:
seems like mypy is complaining

```suggestion
        docstring: str | None = ast.get_docstring(node, clean=False)
```